<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountAddDialog</class>
 <widget class="QDialog" name="accountAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Account</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="1">
    <widget class="QLineEdit" name="savingsInterestRate">
     <property name="placeholderText">
      <string>3.00</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="accountAddTypeLabel">
     <property name="text">
      <string>Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="accountAddBalanceLabel">
     <property name="text">
      <string>Initial Balance ($):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="accountAddIbanLabel">
     <property name="text">
      <string>IBAN:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="accountAddOwnerLabel">
     <property name="text">
      <string>Owner PIN:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="checkingMonthlyFeeLabel">
     <property name="text">
      <string>Monthly Fee ($):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="creditGracePeriodLabel">
     <property name="text">
      <string>Grace Period (days):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="creditAnnualRateLabel">
     <property name="text">
      <string>Annual Rate (%):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="debitTransactionLimitLabel">
     <property name="text">
      <string>Transaction Limit ($):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="debitAtmLimitLabel">
     <property name="text">
      <string>ATM Withdrawal Limit ($):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="savingsInterestRateLabel">
     <property name="text">
      <string>Interest Rate (%):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="savingsMinimumBalanceLabel">
     <property name="text">
      <string>Minimum Balance ($):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="certificateDepositTermLabel">
     <property name="text">
      <string>Deposit Term (months):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="certificateWithdrawalPenaltyLabel">
     <property name="text">
      <string>Withdrawal Penalty (%):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="moneyAllowedTransactionsLabel">
     <property name="text">
      <string>Allowed Transactions:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="moneyTransactionFeeLabel">
     <property name="text">
      <string>Transaction Fee ($):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="certificateWithdrawalPenalty">
     <property name="placeholderText">
      <string>30.00</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLineEdit" name="moneyAllowedTransactions">
     <property name="placeholderText">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="debitAtmLimit">
     <property name="placeholderText">
      <string>2200.00</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="savingsMinimumBalance">
     <property name="placeholderText">
      <string>55.00</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="certificateDepositTerm">
     <property name="placeholderText">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="checkingMonthlyFee">
     <property name="placeholderText">
      <string>5.50</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="accountAddOwner">
     <property name="placeholderText">
      <string>45</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="accountAddIban">
     <property name="placeholderText">
      <string>65</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="accountAddBalance">
     <property name="placeholderText">
      <string>5050.00</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="accountAddType">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>Checking Account</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Credit Card Account</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Debit Card Account</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Savings Account</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Certificate Deposit Account</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Money Market Account</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="creditGracePeriod">
     <property name="placeholderText">
      <string>45</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="creditAnnualRate">
     <property name="placeholderText">
      <string>15.00</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="debitTransactionLimit">
     <property name="placeholderText">
      <string>1100.00</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLineEdit" name="moneyTransactionFee">
     <property name="placeholderText">
      <string>30.00</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="accountAddBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QPushButton" name="accountAddRandomize">
     <property name="text">
      <string>Randomize</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>accountAddBox</sender>
   <signal>accepted()</signal>
   <receiver>accountAddDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accountAddBox</sender>
   <signal>rejected()</signal>
   <receiver>accountAddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
